README.md

```diff
@@ -35,11 +35,8 @@ EventStoreDB is written in a mixture of C#, C++ and JavaScript. It can run on Wi
 ### Build EventStoreDB
 Once you've installed the prerequisites for your system, you can launch a `Release` build of EventStore as follows:
 ```
-dotnet build -c Release src/EventStore.sln -f netcoreapp5.0 -r <runtime identifier>
+dotnet build -c Release src/EventStore.sln
```

Comment:
I think `dotnet build -c Release src` will do the trick as long as there is exactly one `.sln` or `*proj` sufficed file in the folder?

Reply:
I'll try like now :-)